robottelo/api/utils.py

```diff
@@ -130,6 +133,38 @@ def publish_puppet_module(puppet_modules, repo_url, organization_id=None):
     return cv.read()
 
 
+def import_puppet_module(sat, puppet_module_repo=CUSTOM_PUPPET_MODULE_REPOS['generic_1']):
+    """Download, install and import puppet module. This is workaround for 6.10, there is no longer
+    content view
```

Comment:
"...there is no longer way to import puppet modules into a content view." 
(just nitpicking)

Reply:
now it can be used this way, where puppet_class is for example `generic_1`
`default_sat.create_custom_environment(repo=puppet_class)`